experimental/dds/tree2/src/feature-libraries/editable-tree-2/editableTreeTypes.ts

```diff
@@ -190,6 +190,14 @@ export interface TreeField extends Tree<FieldSchema>, Iterable<TreeNode> {
  * @alpha
  */
 export interface MapNode<TSchema extends MapSchema> extends TreeNode {
+	/**
+	 * Returns the number of elements in the map.
```

Comment:
Nit: Since this is a property and not a method, it might make more sense to the end-user to just say
```suggestion
	 * The number of elements in the map.
```

Reply:
Agreed, changed.